### ISSUE TITLE
prediction fail for a specific sequence

### ISSUE THREAD

**Issue by ww2283** [sokrypton/ColabFold#454]
## Current Behavior
A specific sequence fails on local or google colab latest version of ColabFold.
## Steps to Reproduce (for bugs)
Please make sure to reproduce the issue after a "Factory Reset" in Colab. 
If running locally ypdate you local installation `colabfold_batch` to the newest version.
Please provide your input if you can share it. 
Please use the sequence from Uniprot Q3L181, 'MPRVKLGTQGLEVSKLGFGCMGLSGDYNDALPEEQGIAVIKEAFNCGITFFDTSDIYGENGSNEELLGKALKQLPREKIQVGTKFGIHEIGFSGVKAKGTPDYVRSCCEASLKRLDVDYIDLFYIHRIDTTVPIEITMGELKKLVEEGKIKYVGLSEASPDTIRRAHAVHPVTALQIEYSLWTRDIEDEIVPLCRQLGIGIVPYSPIGRGLFAGKAIKESLPENSVLTSHPRFVGENLEKNKQIYYRIEALSQKHGCTPVQLALAWVLHQGEDVVPIPGTTKIKNLHNNVGALKVKLTKEDLKEISDAVPLDEVAGESIHEVIAVTNWKFANTPPLK'.
## ColabFold Output (for bugs)
Please make sure to also post the complete ColabFold output. You can use gist.github.com for large output.
```
Downloading alphafold2 weights to .: 100%|██████████| 3.47G/3.47G [00:25<00:00, 144MB/s]
2023-05-24 15:28:06,480 Running on GPU
2023-05-24 15:28:06,890 Found 5 citations for tools or databases
2023-05-24 15:28:06,891 Query 1/1: Q3L181_8c443 (length 337)
COMPLETE: 100%|██████████| 150/150 [elapsed: 00:02 remaining: 00:00]
2023-05-24 15:28:13,664 Setting max_seq=512, max_extra_seq=5120
2023-05-24 15:29:18,365 Could not predict Q3L181_8c443. Not Enough GPU memory? INTERNAL: cublas error
2023-05-24 15:29:18,367 Done
```
## Context
Providing context helps us come up with a solution and improve our documentation for the future.

## Your Environment
Include as many relevant details about the environment you experienced the bug in.
* Git commit used
* If you run it on a local system. Please add the server specifications
* Operating system and version:

**Comment by sokrypton** [comment]
I added a patch that should hopefully fix this. Can you try again:
https://colab.research.google.com/github/sokrypton/ColabFold/blob/main/AlphaFold2.ipynb

**Comment by sokrypton** [comment]
This fix hasn't been pushed for local installation yet. (Are you seeing same error for local install?)

(reopening issue until we fix for local installations)

**Comment by ww2283** [comment]
yes, I originally came across this problem from local install then tested it via google colab.
